component/Cards/index.html

```diff
@@ -0,0 +1,231 @@
+                <li class="nav-li"> <a href="/component/Badges/index.html" class="nav-a ">Badges</a></li>
+                <li class="nav-li"> <a href="/component/Buttons/index.html" class="nav-a ">Button</a></li>
+                <li class="nav-li"> <a href="/component/Cards/index.html" class="nav-a ">Card</a></li>
+                <li class="nav-li"> <a href="/component/Images/index.html" class="nav-a ">Image</a></li>
+                <li class="nav-li"> <a href="/component/Input/index.html" class="nav-a ">Input</a></li>
+                <li class="nav-li"> <a href="" class="nav-a ">Alerts</a></li>
+                <li class="nav-li"> <a href="" class="nav-a ">Avatar</a></li>
+                <li class="nav-li"> <a href="" class="nav-a ">Alerts</a></li>
+            </ul>
+        </div>
+    </div>
+    <div class="card-body">
+        <h2 class="heading">Cards</h2>
+        <h5 class="card-content">The card component can contain a variety of content, including a heading, image, content, buttons etc.
+        </h5>
+
+        <!-- CARD BASIC WITH RIBBON-BADGE -->
+        <h4 class="btn-subheading">> Basic Card with Badge</h4>
+        <div class="card-align">
+            <div class="card-basic ">
+                <div class="card-txt-container">
+                    <div class="ribbon">
+                        <span class="ribbon__content">FRESH ARRIVAL</span>
+                    </div>
+                    <img class="basic-img" src="/images/headphones.jpg">
+                    <div class="txt-margin">
+                        <div>
+                            <h2 class="card-heading">Headphones</h2>
+                            <h3 class="card-subheading">Rs. 9,600 <span class="card-txt-red"
+                                    style="font-size: 12px; text-decoration:line-through">Rs. 15,999 </span><span
+                                    class="card-txt-secondary">(40% OFF)</span> </h3>
+                        </div>
+                        <p class="card-content-txt">xxxxx
+                        </p>
+                    </div>
+
+                </div>
+                <div class="btn-container">
+                    <div class="btn-standard-container">
+                        <button class="btn-standard btn-std-primary">Read</button>
+                        <button class="btn-outline btn-out-secondary">Bookmark</button>
+                    </div>
+                    <div class="icon-padding">
+                        <button class="btn-icon btn-icon-primary"><i class="fas fa-shopping-cart"></i></button>
+                        <button class="btn-icon btn-icon-primary"><i class="far fa-heart"></i></button>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+
+        <!-- HEADER CARD WITH BADGE -->
+        <div style="height: 30px;"></div>
+        <h4 class="btn-subheading">> Header Card with Badge</h4>
```

Comment:
Remove the extra ">"

Reply:
Extra '>' was for defining a bullet point.